spec/views/events/_apply_button_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "events/_apply_button", type: :view do
+  before(:each) do
+    @event = assign(:event, FactoryGirl.create(:event))
```

Comment:
I'd suggest to use the trait :in_application_phase.
I know that an event is per default in application phase, but if this changes, those tests loose some of their value